### PR TITLE
[s] set all bitstore file type as text/plain refs - #273

### PR DESCRIPTION
**Pull request by subhankarb**

### CODE REVIEW

File: app/package/models.py [hunk 1]
```diff
@@ -122,16 +122,14 @@ def build_s3_object_url(self, domain_name, path):
...
-                                        file_type='binary/octet-stream'):
+                                        acl='public-read'):
         """
         This method produce required data to upload file from client side
         for uploading data at client side
         :param path: The relative path of the object
         :param md5: The md5 hash of the file to be uploaded
 
         :param acl: The object ACL default is public_read
-        :param file_type: The type of the file, default is binary/octet-stream
```
Comment (rufuspollock) [inline app/package/models.py]: Agree on simple removal but think you need to add something to docstring to indicate that file_type will always be set to text/plain and why so future devs know why we did this (and don't refer to the issue - just write it out briefly here).

File: app/package/models.py [hunk 2]
```diff
@@ -122,16 +122,14 @@ def build_s3_object_url(self, domain_name, path):
                    key=self.build_s3_key(path))
 
     def generate_pre_signed_post_object(self, path, md5,
```
Comment (rufuspollock) [inline app/package/models.py]: Is there no change to a controller to reflect this change? Surely the API docs need to change?

Also what about tests?

File: app/package/models.py (hunk 1 above)
Comment (subhankarb) [inline app/package/models.py]: FIXED

File: app/package/models.py (hunk 2 above)
Comment (subhankarb) [inline app/package/models.py]: The filetype was not under test. Added test for that.

Comment (rufuspollock) [inline app/package/models.py]: @subhankarb you have not answered the first question ;-) - i.e. 

> Is there no change to a controller to reflect this change? Surely the API docs need to change?

Comment (subhankarb) [inline app/package/models.py]: @rufuspollock The api docs was 
```
parameters:
        - in: body
          name: data
          type: map
          required: true
          description: publisher name and package name, and file details
```
So there was not mention for filetype.

File: app/package/models.py (hunk 1 above)
Comment (rufuspollock) [inline app/package/models.py]: Can you go add the comment about Why please ...

Comment (subhankarb) [inline app/package/models.py]: @rufuspollock fixed

**Pull request merged**